nautobot/ipam/views.py

```diff
@@ -818,6 +840,14 @@ def post(self, request):
             },
         )
 
+    def assign_ipaddresses_to_interface(self, pks, request):
+        ip_addresses = IPAddress.objects.filter(pk__in=pks)
+        interface_model = Interface if "interface" in request.GET else VMInterface
+        interface_id = request.GET.get("interface") or request.GET.get("vminterface")
+        interface = interface_model.objects.get(id=interface_id)
```

Comment:
Relatedly, probably need an except handler for for the case of a DoesNotExist exception?

Reply:
This case has been handled by the `dispatch` method